src/h3lib/lib/localij.c

```diff
@@ -183,10 +183,8 @@ int h3ToLocalIjk(H3Index origin, H3Index h3, CoordIJK* out) {
         if (originOnPent) {
             int originLeadingDigit = _h3LeadingNonZeroDigit(origin);
 
-            if ((isResClassIII(res) &&
-                 FAILED_DIRECTIONS_III[originLeadingDigit][dir]) ||
-                (!isResClassIII(res) &&
-                 FAILED_DIRECTIONS_II[originLeadingDigit][dir])) {
```

Comment:
This part fixes #163. The change may exclude valid cases, but I think it's okay to be more conservative here. Would like feedback from @isaacbrodsky if available on how the Class III check was expected to work, but I don't think that has to block this change.

Reply:
Definitely want to defer to @isaacbrodsky on this piece, but my gut feeling of applying direction blocks for both hexagon orientations is not the right solution. Either these two should be merged (because there is no difference in reality), or the algorithm behind the `FAILED_DIRECTIONS_*` lookup tables needs to be debugged more closely to see why it missed a path.

Reply:
Since @isaacbrodsky can't look into it at the moment, could we add a comment to revisit this problem in the future? (The only downside is that it might prevent usage of this new function in situations it would be valid in, but since there is no general-purpose fallback here even for cases where this would have failed, that doesn't really change the perceptible end-user behavior.)

Reply:
Makes sense, I will add a TODO comment here explaining what was removed.